[2025-03-29T14:55:55.363+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-29T14:55:55.727+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fist_dag_v5.first_task scheduled__2024-12-25T00:00:00+00:00 [queued]>
[2025-03-29T14:55:55.882+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fist_dag_v5.first_task scheduled__2024-12-25T00:00:00+00:00 [queued]>
[2025-03-29T14:55:55.886+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2025-03-29T14:55:56.049+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): first_task> on 2024-12-25 00:00:00+00:00
[2025-03-29T14:55:56.107+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=284) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-29T14:55:56.110+0000] {standard_task_runner.py:72} INFO - Started process 309 to run task
[2025-03-29T14:55:56.139+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'fist_dag_v5', 'first_task', 'scheduled__2024-12-25T00:00:00+00:00', '--job-id', '169', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp7uthf5da']
[2025-03-29T14:55:56.173+0000] {standard_task_runner.py:105} INFO - Job 169: Subtask first_task
[2025-03-29T14:55:56.545+0000] {task_command.py:467} INFO - Running <TaskInstance: fist_dag_v5.first_task scheduled__2024-12-25T00:00:00+00:00 [running]> on host ba06481f48a9
[2025-03-29T14:55:57.267+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='fist_dag_v5' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2024-12-25T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-25T00:00:00+00:00'
[2025-03-29T14:55:57.294+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-29T14:55:57.585+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-29T14:55:57.600+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is the first task!']
[2025-03-29T14:55:57.795+0000] {subprocess.py:99} INFO - Output:
[2025-03-29T14:55:57.802+0000] {subprocess.py:106} INFO - hello world, this is the first task!
[2025-03-29T14:55:57.826+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-29T14:55:57.950+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-29T14:55:57.953+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=fist_dag_v5, task_id=first_task, run_id=scheduled__2024-12-25T00:00:00+00:00, execution_date=20241225T000000, start_date=20250329T145555, end_date=20250329T145557
[2025-03-29T14:55:59.143+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-29T14:55:59.278+0000] {taskinstance.py:3895} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2025-03-29T14:55:59.312+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
